wavenet.py

```diff
@@ -20,39 +20,40 @@ def _create_dilation_layer(self, input_batch, layer_index, dilation):
         '''Adds a single causal dilated convolution layer.'''
 
         # The filter widths can be configured as a hyperparameter.
-        wf = tf.Variable(tf.truncated_normal(
-            [1, self.filter_width, 256, 256],
+        weights_filter = tf.Variable(tf.truncated_normal(
+            [1, self.filter_width, self.channels, self.channels],
```

Comment:
The DeepMind paper doesn't really go into detail about how to treat the channel dimension, so I've simply left it the same throughout the network.
Changing it throughout could definitely make sense, but we should add this at a later point.